apps/knockdog/src/features/kindergarten-basic/ui/LocationMap.tsx

```
@@ -2,9 +2,10 @@
 
 import { useRef } from 'react';
 import { IconButton } from '@knockdog/ui';
-import { MapView } from '@features/map';
+import { Map, Marker } from '@knockdog/react-naver-map';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)